test/client/api.js

```diff
@@ -734,6 +734,55 @@ module.exports = config => {
       })
   }
 
+  ClientApi.prototype.createTotpToken = function (sessionTokenHex) {
+    return tokens.SessionToken.fromHex(sessionTokenHex)
+      .then((token) => {
+        return this.doRequest(
+          'POST',
+          this.baseURL + '/totp/create',
+          token,
+          {}
+        )
+      })
+  }
+
+  ClientApi.prototype.deleteTotpToken = function (sessionTokenHex) {
+    return tokens.SessionToken.fromHex(sessionTokenHex)
+      .then((token) => {
+        return this.doRequest(
+          'POST',
+          this.baseURL + '/totp/destroy',
+          token,
+          {}
+        )
+      })
+  }
+
+  ClientApi.prototype.checkTotpTokenExists = function (sessionTokenHex) {
+    return tokens.SessionToken.fromHex(sessionTokenHex)
+      .then((token) => {
+        return this.doRequest(
+          'GET',
+          this.baseURL + '/totp/exists',
+          token
+        ).bind(this)
+      })
+  }
+
+  ClientApi.prototype.checkTotpCode = function (sessionTokenHex, code) {
```

Comment:
Would `verify...` be a more appropriate verb than `check...` here, because we're updating state in the db?